apps/drupal-default/particle_theme/templates/block/block--utc-hero-video--full-center.html.twig

```diff
@@ -0,0 +1,153 @@
+{#
+/**
+ * @file Video Hero!
+ * Implements a video hero block in dark blue
+ *
+ *
+ * Available variables:
+ */
+ *
+ * This template pulls in the variables to add to the cards. 
+ * References the block--utc-card-base.html.twig to create the individual cards in this grid.
```

Comment:
Is this comment relevant? I guess this was copied from cards, but I don't think this does mutiple hero or cards?

Reply:
You are correct. It's not relevant. Picked up from previous hero block template and missed on my updates. Will fix both sets.